Arrowgene.Ddon.Shared/Model/Enemy.cs

```diff
@@ -68,6 +67,7 @@ public Enemy(Enemy enemy)
         public uint Experience { get; set; }
         public DropsTable DropsTable { get; set; }
         public bool NotifyStrongEnemy { get; set; }
+        public uint PPDrop {get; set; }
```

Comment:
nitpick: a space between the { and get;

Reply:
I have resolved each of these. I have also explicitly tested it on a server with two enemies, which I had not done before. I set two goblins to have PP drops of 123 and 456. Please check the PR description and you will see an in-game screeenshot demonstrating the PP were dropped. I can also confirm that loading the server with the old JSON (no PPDrop parameter is contained therein) loaded in the expected manner and started the server successfully, though I didn't enter the game and check the PP being dropped.

Reply:
the space seems to be missing still 😛

Reply:
Fixed - for real this time!